diff --git a/vendor/meta/module-catalog/Plugin/FacebookCatalogUpdateFullReindexPlugin.php b/vendor/meta/module-catalog/Plugin/FacebookCatalogUpdateFullReindexPlugin.php
--- a/vendor/meta/module-catalog/Plugin/FacebookCatalogUpdateFullReindexPlugin.php
+++ b/vendor/meta/module-catalog/Plugin/FacebookCatalogUpdateFullReindexPlugin.php
@@ -120,10 +120,10 @@
         }

         $changelogWalkerFactory = $this->objectManager->create(
-            \Magento\Framework\Mview\View\ChangelogBatchWalkerFactory::class // @phpstan-ignore-line
+            \Magento\Framework\Mview\View\ChangeLogBatchWalkerFactory::class // @phpstan-ignore-line
         );
         return $changelogWalkerFactory->create(
-            \Magento\Framework\Mview\View\ChangelogBatchWalker::class // @phpstan-ignore-line
+            \Magento\Framework\Mview\View\ChangeLogBatchWalker::class // @phpstan-ignore-line
         );
     }

